src/sonic-yang-models/yang-models/sonic-vnet.yang

```diff
@@ -0,0 +1,100 @@
+module sonic-vnet {
+    yang-version 1.1;
+    namespace "http://github.com/sonic-net/sonic-vnet";
+    prefix svnet;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    import sonic-vxlan {
+        prefix svxlan;
+    }
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "VNET YANG Module for SONiC OS";
+
+    revision 2023-04-25 {
+        description
+            "First revision.";
+    }
+
+    container sonic-vnet {
+
+        container VNET {
+
+            description "config db VNET table";
+
+            list VNET_LIST {
+
+                key "name";
+                
+                leaf name {
+                    type string;
```

Comment:
Can you add description?
Do we have any pattern for name?

Reply:
name can be any type of string. we dont have a specific requirement that it start with vnet*

Reply:
description added.